backend/tests/manual/test-token-expiration.sh

```
@@ -0,0 +1,165 @@
+  3) PAYLOAD="${PAYLOAD}=" ;;
+esac
+
+# Decode base64 (requires base64 command)
+if command -v base64 &> /dev/null; then
+  DECODED=$(echo "$PAYLOAD" | base64 -d 2>/dev/null || echo "$PAYLOAD" | base64 -D 2>/dev/null)
+  EXP_TIME=$(echo "$DECODED" | grep -o '"exp":[0-9]*' | cut -d':' -f2)
+  IAT_TIME=$(echo "$DECODED" | grep -o '"iat":[0-9]*' | cut -d':' -f2)
+  
+  if [ -n "$EXP_TIME" ] && [ -n "$IAT_TIME" ]; then
+    EXPIRATION_SECONDS=$((EXP_TIME - IAT_TIME))
+    EXPECTED_SECONDS=900  # 15 minutes = 900 seconds
```

Shellcheck (0.11.0)

[warning] EXPECTED_SECONDS appears unused. Verify use (or export if used externally).

(SC2034)